### ISSUE TITLE
Compass calibration: simulate electrical current influence

### ISSUE THREAD

**Issue by hbeni** [HHS81/c182s#521]
The compass calibration card was made with „avionics on“.
It is influenced by both, electrical power to the avionics and also the metal of the plane itself.
It would be cool, if switching on/off of the avionics would be having a simulated effect.

- [x] determine if compass correction card includes only the avionics-on error
- [x] Implement error difference avionics on/off 
- [x] Correct the cockpit card - 90° shows 96 correction, but its actually 4° by the numbers we have configured

- - -
Ref:
- https://www.cfinotebook.net/notebook/avionics-and-instruments/magnetic-compass
- https://www.flight-study.com/2021/04/compass-systems-flight-instruments.html?m=1
- https://www.faa.gov/documentlibrary/media/advisory_circular/ac_43-215.pdf
- https://www.youtube.com/watch?v=WC-iOlckdf4 (Compass errors explained)

C172 also affected, sister PR: https://github.com/c172p-team/c172p/pull/1462

**Comment by TheFGFSEagle** [comment]
If no one else can, I guess I could get a calibration on a real aircraft, at the nearest airfield EDRO - but 1. I most probably won't go there again before april / may (it's too cold right now ! 🧊 ), and 2. it probably won't be from a C182 since the one that is stationed there is flown only rarely ... no idea if that would be of any help.

**Comment by hbeni** [comment]
We already have a calibration, just not the „elec off/only metal influence“. No worries

**Comment by hbeni** [comment]
Reading this: https://www.cfinotebook.net/notebook/avionics-and-instruments/magnetic-compass
Explains it somewhat.
So the correction card shows the _remaining_ error after adjusting the compensator (ie the variable error depending on aircraft orientation, which affects both, the bare metal influence and avionics elecrical fields)

**Comment by hbeni** [comment]
Got another piece of information here, an example of the influence of radios:
https://www.flight-study.com/2021/04/compass-systems-flight-instruments.html?m=1

![IMG_6613](https://github.com/HHS81/c182s/assets/13608602/8d485821-87d5-40e3-92b7-a800d4d51a54)

in this case the radios counter-compensate, (radio off=more error)

**Comment by hbeni** [comment]
FAA document describes the provedure how the compensator is adjusted (p. 5f).
Tldr; half if the error will remain, resulting in an averaged errornous reading (what is the „remaining error“ noted on the correction card).

https://www.faa.gov/documentlibrary/media/advisory_circular/ac_43-215.pdf
